File: revisao/lista/Lista.go
package main
import "fmt"
import "container/list"

func main(){
	nomes := list.New()
	nomes.PushBack("Josimar")
	nomes.PushBack("Angélica")
	for e := nomes.Front(); e != nil; e = e.Next() {
        fmt.Println(e.Value)
    }
}

File: revisao/lista/Lista.java
import java.util.ArrayList;
import java.util.Iterator;
import java.util.List;


public class Lista {
    public static void main(String[] args) {
        List<String> nomes = new ArrayList<String>();
        nomes.add("Josimar ");
        nomes.add("Angélica ");
/*         System.out.println("\nFor clássico:");
        for(int i=0; i<nomes.size();i++){
            System.out.print(nomes.get(i));
        }
        System.out.println("\nIterator:");
        Iterator i = nomes.iterator();
        while (i.hasNext()){
            System.out.print(i.next());
        }
        System.out.println("\nfor-each:");
        for(String nome: nomes){
            System.out.print(nome);
        } 
        System.out.println("\nforEach():");*/
        nomes.forEach(System.out::print);
    }
}
